modules/utxo-lib/test/integration_local_rpc/generate/main.ts

```diff
@@ -113,19 +113,32 @@ async function run(network: Network) {
   }
 }
 
-async function main() {
+async function main(args: string[]) {
+  const allowedNetworks = args.map((name) => {
+    const network = utxolib.networks[name];
+    if (!network) {
+      throw new Error(`invalid network ${name}`);
+    }
+    return getMainnet(network);
```

Comment:
Also curious here as to why we want the mainnet network.

Reply:
we normalize to mainnet so that it works regardless if you use `bitcoingoldTestnet` or just `bitcoingold` as the argument